### DISCUSSION TITLE
Using .tar.gz linux driver

### DISCUSSION THREAD

**Discussion by rodrigooeiras** [OpenTabletDriver/OpenTabletDriver#1434] [Q&A] [answered]
Hello, 

I was trying the linux driver, OpenTabletDriver.linux-x64.tar.gz, but I couldn't. 
How to install?

I tried:
"# Reload the systemd user unit daemon
systemctl --user daemon-reload
# Enable and start the user service
systemctl --user enable opentabletdriver --now"

I got:
"Failed to enable unit: Unit file opentabletdriver.service does not exist."

obs: I installed dotnet by snap. 
dotnet --list-sdks
5.0.400 [/snap/dotnet-sdk/137/sdk]

Thanks in advance.

**Comment by InfinityGhost** [comment]
What distribution are you running? It's likely that we have a package that you can use that would avoid all of these issues.

**Reply by jamesbt365** [reply]
If this isn't the case then you have to go ahead and make the service youself.

**Comment by gonX** [comment, marked as answer]
To use the .tar.gz archive, you need to execute `otd-daemon` to launch the driver, and make sure that udev rules are installed and configured correctly for your tablet. These are included with the tar.gz release archives.
